wrappers/javascript/anoncreds-shared/src/Anoncreds.ts

```diff
@@ -39,6 +39,7 @@ export interface Anoncreds {
   version(): string
 
   getCurrentError(): string
+  setDefaultLogger(): void
```

Comment:
Is there a way in these new components to set a custom logger like we did in indy-sdk to log in our language of choice?

Reply:
We never implemented that method for React Native, only Node.JS. I think it should be possible, but not sure how to do it currently. We thought defaultLogger would be enough for now and we can add the custom logger later on

Reply:
I didn't do that recently but I remember that in NodeJS I needed to set environment variable `RUST_LOG=anoncreds:loglevel` in addition to a call to set_default_logger.  In React Native this can also be achieved so Rust traces can be seen in Android logcat or something?